src/server/routers/pokemon.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express");
```

Comment:
Will this be removed once the db is populated? Or should we leave it in case we decide we need other data?